doc/languages-frameworks/qt.section.md

```diff
@@ -25,12 +25,14 @@ stdenv.mkDerivation {
 
 The same goes for Qt 5 where libraries and tools are under `libsForQt5`.
 
-Any Qt package should include `wrapQtAppsHook` in `nativeBuildInputs`, or explicitly set `dontWrapQtApps` to bypass generating the wrappers.
+Any Qt package should include `wrapQtAppsHook` (or `wrapQtAppsNoGuiHook`) in `nativeBuildInputs`, or explicitly set `dontWrapQtApps` to bypass generating the wrappers.
 
 ::: {.note}
-Qt 6 graphical applications should also include `qtwayland` in `buildInputs` on Linux (but not on platforms e.g. Darwin, where `qtwayland` is not available), to ensure the Wayland platform plugin is available.
 
-This may become default in the future, see [NixOS/nixpkgs#269674](https://github.com/NixOS/nixpkgs/pull/269674).
+`wrapQtAppsHook` will propagate `qtwayland` to ensure the Wayland platform plugin is available, which is useful for graphical applications.
+
+`wrapQtAppsNoGuiHook` will not propagate any qt components to reduce closure size for purely command-line applications.
```

Comment:
Qt

Reply:
Also, maybe "anything but qtbase"? qtbase is technically a Qt component

Reply:
"additional qt components", I propose.

Reply:
> Also, maybe "anything but qtbase"? qtbase is technically a Qt component

Well it does not propagate that currently?